webpack/JobInvocationDetail/JobInvocationSystemStatusChart.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ChartDonut, ChartLabel, ChartLegend } from '@patternfly/react-charts';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  FlexItem,
+  Text,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './JobInvocationDetail.scss';
+
+const JobInvocationSystemStatusChart = ({
+  data,
+  isAlreadyStarted,
+  formattedStartDate,
+}) => {
+  const {
+    succeeded,
+    failed,
+    pending,
+    cancelled,
+    total,
+    total_hosts: totalHosts,
+  } = data;
+  const chartData = [
+    { title: __('Succeeded:'), count: succeeded, color: '#3E8635' },
+    { title: __('Failed:'), count: failed, color: '#C9190B' },
+    { title: __('In Progress:'), count: pending, color: '#2B9AF3' },
+    { title: __('Canceled:'), count: cancelled, color: '#6A6E73' },
+  ];
+  const chartDonutTitle = () => {
+    if (total > 0) return `${succeeded.toString()}/${total}`;
+    if (totalHosts > 0) return `0/${totalHosts}`;
+    return '0';
+  };
+
+  return (
+    <>
+      <FlexItem className="chart-donut">
+        <ChartDonut
+          constrainToVisibleArea
+          data={chartData.map(d => ({
+            label: `${d.title} ${d.count} hosts`,
+            y: d.count,
+          }))}
+          colorScale={total > 0 ? chartData.map(d => d.color) : ['#8A8D90']}
+          subTitle={__('Systems')}
+          title={chartDonutTitle}
+          titleComponent={<ChartLabel />}
+          subTitleComponent={<ChartLabel />}
+          padding={{
+            bottom: 0,
+            left: 0,
+            right: 0,
+            top: 0,
+          }}
+          width={105}
+          height={105}
+        />
+      </FlexItem>
+      <FlexItem className="chart-legend">
+        <Text className="legend-title">{__('System status')}</Text>
```

Comment:
I'm not sure if `System` is a fortunate wording in this case.

Reply:
Do you please have any other ideas?

Reply:
https://www.sketch.com/s/9c533bae-e910-4a9c-89a1-fe67b04a84ca/a/mPmrkMJ
its the word from the design as well